easybuild/easyconfigs/c/Catch2/Catch2-3.6.0-GCCcore-13.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'Catch2'
+version = '3.6.0'
+
+homepage = 'https://github.com/catchorg/Catch2'
+description = """A modern, C++-native, header-only,
+ test framework for unit-tests, TDD and BDD
+ - using C++11, C++14, C++17 and later
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/catchorg/Catch2/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['485932259a75c7c6b72d4b874242c489ea5155d17efa345eb8cc72159f49f356']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('CMake', '3.27.6'),
+]
+
+toolchainopts = {'pic': True}
+
+separate_build_dir = True
```

minor nitpick -- for consistency with most other easyconfigs, toolchainopts should be defined after the toolchain. Also, `separate_build_dir` is already default behaviour, so doesn't need to be made explicit.

```suggestion
toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
toolchainopts = {'pic': True}

source_urls = ['https://github.com/catchorg/Catch2/archive/']
sources = ['v%(version)s.tar.gz']
checksums = ['485932259a75c7c6b72d4b874242c489ea5155d17efa345eb8cc72159f49f356']

builddependencies = [
    ('binutils', '2.40'),
    ('CMake', '3.27.6'),
]
```

---

Thank you for pointing that out.
I will try and keep an eye on it in the future